### DISCUSSION TITLE
Visual Git In Sidebar?

### DISCUSSION THREAD

**Discussion by andrewfluck** [zed-industries/zed#6514] [Feature Requests]
Hey y'all! I recently stumbled across this and am enjoying my usage of it so far. I'm trying to sell my team on it. I was curious to know if there was visual git in the sidebar yet? This is probably a big thing that would prevent some folks from jumping on board using it, though, I don't know if I'm searching for the wrong feature?

**Comment by clo4** [comment]
#29 is what you're looking for! Not yet but coming in the future, likely pre 1.0 for mvp

**Reply by andrewfluck** [reply]
Thank you for the answer! I'll keep my eyes on this one!

**Reply by rollemira** [reply]
This would be amazing. The integrated git experience of vscode is great because it wraps a lot of the cli options that make the source control experience so much better. A lot of cool "at a glance" things too. After trying zed for the first time the source control integration was really the only thing I missed about my old editor.

**Comment by JosephTLyons** [comment]
We now have project wide - diffs powered by the multi-buffer and the outline side panel.

All other git-panel related stuff is still coming:

- https://github.com/zed-industries/zed/issues/4367
